.github/workflows/pull_request_build.yml

```diff
@@ -6,12 +6,11 @@
 # http://www.eclipse.org/legal/epl-v20.html
 #
 # SPDX-License-Identifier: EPL-2.0
-name: Antenna Push Build
+name: Antenna Pull Request Build
```

Comment:
Commit message:
- s/travsi/Travis/
- s/gitub/GitHub/

In general, from readin the commit message I would have expected a `.travis.yml` being touched / removed in favor of a GitHub actions YAML, but that's not what seems to be the case in the strict sense. In fact, nothing Travis-related seems to happen in this commit at all.